guides-markdown/wallets.md

```diff
@@ -243,12 +243,26 @@ console.log(jdWalletKey0.getPrivateKey('base58'));
 //cNZfR3NhQ9oCP3pTjvPZETUuTWZo2k6EXtfczvbWyv7FdjMhppvJ
 ```
 
+#### Using an HD path
+
+Sometimes it is handy to work with an HD path. This is an object that contains information on a particular path to a key in the tree hierarchy of the wallet. In particular, a path knows the account, the branch and the index of the relevant key. This is according to [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki), which specifies that every account gets its own subtree of keys. An account has three children (known as branches): the first for receiveing addresses, the second for change addresses and the third for segwit addresses. These three branches can in turn have as many children as needed. These correspond to the aforementioned index and provide the actual keys that are used for transactions.
```

Comment:
Might be worth specifying that branch `2` is for _nested_ segwit addresses only, and this is actually NOT part of BIP44, it's sort of a hack implemented only in the bcoin wallet. Native bech32 Segwit addresses can be derived from the normal change/receive branches as well in bcoin and other BIP44 wallet implementations. Other BIPs like 49 and 84 specify HD path purposes for different types of Segwit addresses and although there is a PR to support these standards, it is not yet merged into bcoin.

Reply:
Fixed (with full disclosure...)